server/sonar-web/src/main/js/apps/organizations/components/OrganizationMembers.js

```diff
@@ -58,23 +60,27 @@ export default class OrganizationMembers extends React.PureComponent {
     }
   }
 
-  handleSearchMembers = (query?: string) => {
+  handleSearchMembers = (query /*: string | void */) => {
     this.props.fetchOrganizationMembers(this.props.organization.key, query);
   };
 
   handleLoadMoreMembers = () => {
     this.props.fetchMoreOrganizationMembers(this.props.organization.key, this.props.status.query);
   };
 
-  addMember = (member: Member) => {
+  addMember = (member /*: Member */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)